test/torchaudio_unittest/transforms/batch_consistency_test.py

```diff
@@ -16,12 +16,10 @@ def assert_batch_consistency(self, transform, batch, *args, atol=1e-8, rtol=1e-5
         n = batch.size(0)
 
         # Compute items separately, then batch the result
-        torch.random.manual_seed(seed)
```

Comment:
Same here, the helper method must run the given function twice with the same seed. So these two seedings in this method have to stay.